src/main/java/Conversation.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Scanner;
+public class Conversation {
+    public Conversation(){
+
+    }
+    public void dividingLine(){
+        System.out.println("\t__________________________________________________");
+    }
+    public void communicate(){
+        CommunicateCaseHandle cch = new CommunicateCaseHandle();
```

Comment:
variable cch's pupose is not clear from the name

Reply:
what name your recommende for this